spec/usage/extend/custom_component_spec.rb

```diff
@@ -0,0 +1,169 @@
+require_relative '../../support/utils'
+include Utils
+
+describe 'Creating custom components', type: :feature, js: true do
+
+  before :all do
+
+    module Components end
+
+    module Pages end
+
+    module Matestack::Ui::Core end
+
+    class ComponentTestController < ActionController::Base
+      layout 'application'
+
+      include Matestack::Ui::Core::ApplicationHelper
+
+      def my_action
+        responder_for(Pages::ExamplePage)
+      end
+
+    end
+
+    Rails.application.routes.append do
+      get '/custom_component_test', to: 'component_test#my_action'
+    end
+    Rails.application.reload_routes!
+
+  end
+
+  it 'by orchestrating existing core components' do
```

Comment:
this tests specifies creating a `custom static component`, I'm missing the `custom dynamic component` spec. This is only covered in the `Actionview` dynamic spec

Reply:
Been there, done that